multimodal/src/autogluon/multimodal/data/infer_types.py

```diff
@@ -345,9 +345,9 @@ def infer_label_column_type_by_problem_type(
             check_missing_values(data=data, column_name=col_name, split="training")
         if valid_data is not None:
             check_missing_values(data=valid_data, column_name=col_name, split="validation")
-        if col_name in column_types and column_types[col_name] == NULL:
```

Comment:
Is this check happening somewhere else? Wonder why removing it

Reply:
We assert `col_name in column_types` in Line 342.